src/guiguts/tools/pptxt.py

```diff
@@ -608,9 +608,12 @@ def weird_characters() -> None:
     line_number = 1
     for line in book:
         # Get list of weirdos on this line. That means any character NOT in the regex.
-        weirdos_list = re.findall(r"[^A-Za-z0-9\s.,:;?!\\\-_—–=“”‘’\[\]\(\){}]", line)
+        weirdos_list = re.findall(r"[^A-Za-z0-9\s.,:;?!&\\\-_—–=“”‘’\[\]\(\){}]", line)
         # Update dictionary with the weirdos from the line.
         for weirdo in weirdos_list:
+            # Skip exceptions
+            if weirdo == "*" and line == "       *" * 5:
```

Comment:
I think it would be better to use a regex, and not assume the # of spaces is always the same.

Reply:
Except this way it would warn you if your tb wasn't standard, eg had some spaces missing.

Reply:
What kind of variation were you thinking of?

Reply:
Probably OK as-is. The spacing between should at least be consistent, but I'm not sure how to specify that in regex.

Reply:
I can do that in a regex, if you specify what spacings you want to allow

Reply:
GG uses 7 spaces on tb conversion, maybe 3 to 10?